### DISCUSSION TITLE
Anthropic error

### DISCUSSION THREAD

**Discussion by Mapping730** [open-webui/open-webui#8117] [General]
## Installation Method

upgraded by doc.openwebui.ui suggested method, 
docker rm -f open-webui
docker pull ghcr.io/open-webui/open-webui:main
docker run -d -p 3000:8080 -v open-webui:/app/backend/data --name open-webui ghcr.io/open-webui/open-webui:main

seemed to work fine, opened local host, logged in and dtat was all ther, i believe

## Environment

- **Open WebUI Version:** to 5.1
- **Ollama (if applicable):**0.5.1 , but i run all models remote

- **Operating System:** windows 10
- **Browser firefox 133

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [ X] I am on the latest version of both Open WebUI and Ollama.
- [ X] I have included the browser console logs.
- [X ] I have included the Docker container logs.
- [X ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

open new chat, said hi, wanted a hey back

## Actual Behavior:

received a session_id error and a Error: HTTP Error 401: {"type":"error","error":{"type":"authentication_error","message":"invalid x-api-key"}}  same key used on another openwebui session.  

## Description

**Bug Summary:**
error upon start, other models work including gemini
not sure, upgraded to 5.1 and this happened,  im also getting session_id errors as well, but i will open another report for that. 

## Reproduction Details

using the anthorpic function, i put in the key i use for every other instance, im 99% sure the key is good.  put the key in the function valve, and save, go to a new conversation and say hi, and error.  using google function as well and works.

## Logs and Screenshots
![image](https://github.com/user-attachments/assets/861d309f-bd0c-465a-b605-3d933514045b)

**Browser Console Logs:**
[console-export-2024-12-26_11-39-15.txt](https://github.com/user-attachments/files/18254149/console-export-2024-12-26_11-39-15.txt)

**Docker Container Logs:**
[DOCKERLOG.txt](https://github.com/user-attachments/files/18254173/DOCKERLOG.txt)

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Incompatible function, using deprecated internal functions, please reach out to the authors.
